### ISSUE TITLE
Evidence Status mgmt

### ISSUE THREAD

**Issue by atulgupta2024** [pucardotorg/dristi#2682]
As part of document submission, there is a new field "status" added to filing TAB on UI, that needs to show the status of artifact (evidence). Following are different possible states/statuses 

Artifact submitted during Case filing
1. Case in pre-admission state - "Pending"
2. Case admitted - "Submitted"
3. Case closed (outcome reached) - no change in status
4. Case abandoned/abated - "Void" OR no change in status

Artifact submitted with Application 
1. Application draft/pending approval - "Pending"
2. Application accepted - "Accepted"
3. Application rejected - "Void"

Artifact submitted via Direct Submission
1. Document pending submission (if possible) - "Pending"
2. Document submitted - "Submitted"
3. Document voided by Judge - "Void"

The "Status" field will not be linked with any workflow status directly. 
We will no longer need the "IsVoid" field and it will be removed. 
A new Master to be created with these possible status values for Artifacts. 

@manimaarans @anirudh-0 @suresh12 @devarajd94 @anshumanth-egov @subhashini-egov @Taherabharmal @mehulseg24 - review and comment. 

cc. @Ramkrishna-egov

**Comment by anshumanth-egov** [comment]
Artifact submitted via Direct Submission : Document pending submission (if possible) - "Pending" 
Could you clarify on if possible?
For direct submission, all cases I know of, document goes directly into Submitted status. 

Case abandoned/abated: we can leave it as is, no change in status

**Comment by atulgupta2024** [comment]
@anshumanth-egov yes, direct document submission the idea is it will go submitted directly but it has an eSign requirement. imaging esign didn't work, or app crashed or something happened, if nothing is saved, then fine, but if we do persist and start off e-sign, we can use the "Pending" status in between.

**Comment by atulgupta2024** [comment]
btw, I just realized, we do have "delete" option in workflows.. so if deleted, shall we mark the status as "Void"?

**Comment by anshumanth-egov** [comment]
> @anshumanth-egov yes, direct document submission the idea is it will go submitted directly but it has an eSign requirement. imaging esign didn't work, or app crashed or something happened, if nothing is saved, then fine, but if we do persist and start off e-sign, we can use the "Pending" status in between.

Yes makes sense.

**Comment by anshumanth-egov** [comment]
> btw, I just realized, we do have "delete" option in workflows.. so if deleted, shall we mark the status as "Void"?

Delete for what? haven't seen that before, didn't think we had that.

**Comment by atulgupta2024** [comment]
delete document submission, delete application submission. the case itself maybe deleted (can accommodate with Void)
